### ISSUE TITLE
[Ruby] Invalid symbol highlighting

### ISSUE THREAD

**Issue by ilnurnasyrov** [sublimehq/Packages#1310]
```ruby
{ 'x': 1 }
```
`'x':` should be highlighted as symbol, but now it looks like string in ST.

**Comment by FichteFoll** [comment]
Can you describe how it looked previously?

**Comment by ilnurnasyrov** [comment]
It always have been the same. Moreover, github, visual studio, atom highlight it the same. But I believe, it should be highlighted as symbol, because.

```ruby
{ 'x': 1 }.keys #=> [:x]
```

**Comment by Thom1729** [comment]
I think you could make a case that while the `'x'` *semantically* represents a symbol, it is *syntactically* a string.

**Comment by ilnurnasyrov** [comment]
no, it is semantically represents symbol and syntactically is a symbol. There only wrong highlighting i believe.

![image](https://user-images.githubusercontent.com/3808127/33009059-85409d9a-cde6-11e7-9833-723979cf5a08.png)

**Comment by Thom1729** [comment]
You make a compelling argument.
